src/ResourceManager/Security/Commands.Security/Cmdlets/Alerts/GetAlerts.cs

```diff
@@ -0,0 +1,108 @@
+using Microsoft.Rest.Azure;
+
+namespace Microsoft.Azure.Commands.Security.Cmdlets.Alerts
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmSecurityAlert", DefaultParameterSetName = ParameterSetNames.SubscriptionScope), OutputType(typeof(PSSecurityAlert))]
+    public class GetAlerts : SecurityCenterCmdletBase
+    {
+        [Parameter(ParameterSetName = ParameterSetNames.ResourceGroupScope, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.ResourceGroupName)]
+        [Parameter(ParameterSetName = ParameterSetNames.ResourceGroupLevelResource, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.ResourceGroupName)]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.ResourceGroupLevelResource, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.ResourceName)]
+        [Parameter(ParameterSetName = ParameterSetNames.SubscriptionLevelResource, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.ResourceName)]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.ResourceGroupLevelResource, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.Location)]
+        [Parameter(ParameterSetName = ParameterSetNames.SubscriptionLevelResource, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.Location)]
+        [ValidateNotNullOrEmpty]
+        [LocationCompleter]
+        public string Location { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.ResourceId, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = ParameterHelpMessages.ResourceId)]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            try
+            {
+                switch (ParameterSetName)
+                {
+                    case ParameterSetNames.SubscriptionScope:
+                        var alerts = SecurityCenterClient.Alerts.ListWithHttpMessagesAsync().GetAwaiter().GetResult().Body;
+                        WriteObject(alerts.ConvertToPSObject(), enumerateCollection: true);
+                        break;
+                    case ParameterSetNames.ResourceGroupScope:
+                        alerts = SecurityCenterClient.Alerts.ListByResourceGroupWithHttpMessagesAsync(ResourceGroupName).GetAwaiter().GetResult().Body;
+                        WriteObject(alerts.ConvertToPSObject(), enumerateCollection: true);
+                        break;
+                    case ParameterSetNames.SubscriptionLevelResource:
+                        SecurityCenterClient.AscLocation = Location;
```

Comment:
this is correct, we would like to change this in the .Net SDK as well because of that
in the meantime, I used this pattern that to have the PS cmdlets changes to a minimum